### PR TITLE
[docs] Installing TestCafe

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
/cc @DevExpress/testcafe-docs

### CODE REVIEW

**Comment by kirovboris** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
Are you sure that

```
testcafe -v
```

is an apt sample to show testcafe 'running'

**Comment by VasilyStrelyaev** [comment]
mmm.... probably it's good... I'm just thinking out loud

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 1]
```diff
@@ -6,15 +6,62 @@ checked: true
...
+
+After that, you can run TestCafe by using the `testcafe` command (see [Usage](#usage)).
+
+```bash
+testcafe -v
+```
+
+### Locally
+
+The following command will install TesCafe into your project directory and save it in the dependencies list.
```
Comment (AlexanderMoskovkin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: typo: The following command will install **TesCafe** into your

**Comment by AlexanderMoskovkin** [comment]
lgtm, just a typo

**Comment by MargaritaLoseva** [comment]
> testcafe -v
> is an apt sample to show testcafe 'running'
You are right. I'll change this.

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 2]
```diff
@@ -6,15 +6,62 @@ checked: true
...
+
+* It makes your project setup easier: running `npm install` in the project directory will automatically install TestCafe as well.
+* Different projects can depend on different versions of TestCafe.
+
+You can run the local version of TestCafe in one of the following ways.
+
+* From the `node_modules` folder.
+
+    ```bash
+    node node_modules/testcafe/bin/testcafe -v
```
Comment (AndreyBelym) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: All locally installed binaries are linked to `node_modules/.bin`. So the preferred way to work with local binary is executing it `./node_modules/.bin/testcafe`. But, maybe we don't really need to point out how to use local binary in CLI: https://github.com/DevExpress/testcafe/issues/640#issuecomment-230416634. \cc @inikulin

**Comment by churkin** [comment]
lgtm

**Comment by MargaritaLoseva** [comment]
fpr

**Comment by VasilyStrelyaev** [comment]
lgtm

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 3]
```diff
@@ -6,15 +6,62 @@ checked: true
...
+
+* It makes your project setup easier: running `npm install` in the project directory will automatically install TestCafe as well.
+* Different projects can depend on different versions of TestCafe.
+
+You can run the local version of TestCafe in one of the following ways.
+
+* From the `node_modules` folder.
+
+    ```bash
+    node node_modules/testcafe/bin/testcafe safari tests/
```
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: This is not how it used usually. You need to add it to [npm scripts](https://docs.npmjs.com/misc/scripts)

File: docs/articles/documentation/using-testcafe/command-line-interface.md [hunk 4]
```diff
@@ -6,15 +6,62 @@ checked: true
 ---
 # Command Line Interface
 
+This topic provides information on how to run TestCafe tests from a command line.
+
+## Installation
```
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/command-line-interface.md]: I don't understand why whole installation topic belongs to command-line-interface. I have strong feeling that it shouldn't be here and more likely should implemented as separate topic. For programmatic usage you will have the same steps. Also we need to reference full installation topic from GS

**Comment by inikulin** [comment]
\r-

**Comment by MargaritaLoseva** [comment]
fpr

File: docs/articles/documentation/using-testcafe/installing-testcafe.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+## Locally
+
+The following command will install TestCafe into your project directory and save it in the dependencies list.
+
+```bash
+npm install --save-dev testcafe
+```
+
+The local installation is preferred if you are going to use TestCafe from a Node.js application (see [Programming Interface](programming-interface/index.html)). This kind of installation has two advantages:
```
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/installing-testcafe.md]: First of all it's prefered for CI scenarios.

**Comment by inikulin** [comment]
\r-

File: docs/articles/documentation/using-testcafe/installing-testcafe.md (hunk 5 above)
Comment (inikulin) [inline docs/articles/documentation/using-testcafe/installing-testcafe.md]: IMHO we shouldn't even mention programming interface here

**Comment by MargaritaLoseva** [comment]
fpr

File: docs/articles/documentation/using-testcafe/programming-interface/index.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ permalink: /documentation/using-testcafe/programming-interface/
 ---
 # TestCafe Programming Interface
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/using-testcafe/programming-interface/index.md]: pls change the name as we discussed, as long as you modify this file

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by MargaritaLoseva** [comment]
fpr

**Comment by inikulin** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by kirovboris** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Pull request merged**